service/src/main/java/bio/terra/workspace/service/resource/controlled/flight/delete/DeleteControlledResourceFlight.java

```diff
@@ -73,6 +54,7 @@ public DeleteControlledResourceFlight(FlightMap inputParameters, Object beanBag)
             userRequest),
         samRetryRule);
 
+    RetryRule gcpRetryRule = RetryRules.cloud();
```

Comment:
nit: `final`? ditto other classes (some use `final`, some don't). would be good to be consistent.

Reply:
In the style guide it's optional for local declarations. I like it everywhere because it calls out assignments and I don't have to worry about reassignment, but some other folks aren't as convinced.